### PR TITLE
ACC-832 Carroll County Fix Max 

### PR DESCRIPTION
**Pull request by Aidancs**
refs: https://spurwork.atlassian.net/browse/ACC-832

### CODE REVIEW

**Review by stevespur — APPROVED** [review]

File: tests/Unit/Countries/US/Kentucky/V20190101/local/CarrollCountyTest.php [hunk 1]
```diff
@@ -109,12 +109,28 @@ public function provideTestData(): array
...
                     ->setYtdLiabilitiesInCents(5000000)
-                    ->setExpectedAmountInCents(100)
+                    ->setExpectedAmountInCents(0)
+                    ->build()
+            ],
+            'over max tax' => [
+                $builder
+                    ->setWagesInCents(10000)
+                    ->setYtdWagesInCents(0)
+                    ->setYtdLiabilitiesInCents(5000100)
```
Comment (stevespur) [inline tests/Unit/Countries/US/Kentucky/V20190101/local/CarrollCountyTest.php:132]: this shouldn't be right should it? 5m cents for $50k shoudn't hit the max should it?

Comment (Aidancs) [reply tests/Unit/Countries/US/Kentucky/V20190101/local/CarrollCountyTest.php:132]: Max is already hit there. The YtdLiabilities is what has been paid already and it is over max.

Comment (stevespur) [reply tests/Unit/Countries/US/Kentucky/V20190101/local/CarrollCountyTest.php:132]: ah, right, that's amount paid not wages earned :duh:

Comment (Aidancs) [reply tests/Unit/Countries/US/Kentucky/V20190101/local/CarrollCountyTest.php:132]: It is confusing which goes to which, lots of dumps to figure out the values

**Review by rmcdaniel — APPROVED** [review]

**Pull request merged**